.github/workflows/commit_syntax.yml

```diff
@@ -1,20 +1,10 @@
 name: 'Commit Message Check'
```

Comment:
@erichulburd I removed the `pull_request_target` because it was causing problems. Seems to work well now with these changes, so let me know if there are any concerns.